test/cmdln_parser_test.py

```diff
@@ -1,73 +1,70 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 
 # A tests for scout
 #
 
+from copy import copy
+from optparse import OptionValueError
 import sys
+import unittest
 sys.path.insert(0, '../scout/')
-import __init__ as scout
+import scout
+
 if not hasattr(scout, 'ScoutCore'):
     raise ImportError('No main scout module found')
 
-import sys, os
-
-from copy import copy
-from optparse import OptionValueError
-import unittest
 
 class CoreOptionParserTestCase(unittest.TestCase):
 
     def setUp(self):
-
         self.argv_bak = copy(sys.argv)
-        sys.argv=['scout']
+        sys.argv = ['scout']
 
-        self.modules = (("python" , "Search for python modules"), ("java", "Search for java classes"))
+        self.modules = (('python', 'Search for python modules'),
+                        ('java', 'Search for java classes'))
         self.formats = ('table', 'xml', 'csv')
         self.parser = scout.CoreOptionParser(self.formats, self.modules)
 
     def tearDown(self):
-
         sys.argv = copy(self.argv_bak)
 
     def testInit(self):
-
-        self.assertEqual(self.parser.prog, "scout")
-        self.assertEqual(self.parser.format, "table")
+        self.assertEqual(self.parser.prog, 'scout')
+        self.assertEqual(self.parser._format, 'table')
 
     def testNoArgs(self):
-
         try:
-            ret = self.parser.parse_args()
+            self.parser.parse_args()
         except scout.HelpOptionFound:
             pass
         else:
-            self.fail("The HelpOptionFound have to be raised for no defined argument")
+            self.fail('The HelpOptionFound have to be raised '
+                      'for no defined argument')
```

Comment:
You can simplify this test to

```python
    def testNoArgs(self):
        with self.assertRaises(scout.HelpOptionFound, 'The HelpOptionFound have to be raised for no defined argument')
            self.parser.parse_args()
```

Same for other tests that expect an exception.

Reply:
Tests are broken. I agree that this is a better approach but for this commit I only want to port the same code base.

Reply:
+1 for @cboltz change, but again, not a big deal.